test-utils/src/templates/schema_derivation_test_body_template

```diff
@@ -1,16 +1,13 @@
+
```

Comment:
nit: is this intentional?

Reply:
The newline at the beginning is intentional, although I realize now it's probably not the best way to achieve this. My goal was to put a space between each function in the generated files (I was looking at them a lot during development, so for my own sanity it was useful to break them apart a bit).

I can either change this to be double newlines at the end of the file, or I can just remove this since now we're back to the case where it's unlikely someone needs to read the generated files. Or I can leave this as-is, if you don't care enough either way.